packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# 51.14.0 - May 21, 2021
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Enable diff detection for folders ([PR #3243](https://github.com/forcedotcom/salesforcedx-vscode/pull/3243))
```

Comment:
@sbudhirajadoc whenever you get a chance to review - let me know if you have any questions! Wasn't sure if we had some more details we wanted to include for Diff Detection?

Reply:
The language in the change log is not consistent so we could go either way at this point, but did you want to say "Enabled", maybe?

Reply:
Yeah “Enabled” sounds good to me

Reply:
@sbudhirajadoc Actually it looks like we've been using present tense for the other changelog entries, so should we leave it as `Enable`?

Reply:
I looked at older updates and we seem to have a hodge-podge of both present and past tenses. Past tense is preferred (Fixed, Updated, Resolved, Added, Improved etc). I am fine with leaving it in present tense. I will work on a template that we can use in the future logs.

Reply:
Sounds good to me, a template would be helpful!